release-notes-5.0.0.md

```diff
@@ -4,5 +4,7 @@
 ${version-number}
 
 #### New Features
```

Comment:
@dermot-hardy Should I list this as a breaking change instead of new feature?

Reply:
It is both, and it's fine to list it twice in two sections.  You'd be hard pushed to actually see the break though.  You'd need to have a .NET 8 project that uses functionality that has been removed in .NET 9 and you'd need to have tests exercising that functionality to see the break.